conf/config-default.yaml

```diff
@@ -174,6 +174,11 @@ nginx_config:                     # config for render the template to generate n
   #  - TEST_ENV
 
   stream:
+    enable_access_log: true         # enable access log or not, default true
```

Comment:
The access_log in stream is false by default: https://nginx.org/en/docs/stream/ngx_stream_log_module.html#access_log

Reply:
fixed